README.md

```diff
@@ -1,29 +1,102 @@
 # testcafe-aurelia-selectors
 
-[Tescafe](https://github.com/DevExpress/testcafe) selector extentions for [Aurelia](https://github.com/aurelia/framework) apps.
-
-## Avalible selector extentions
-- byValueBind
-- byShowBind
-- byCheckedBind
-- byFocusBind
-- byDbClickDelegate
-- byClickDelegate
+This plugin provides selector extensions that make it easier to test Aurelia applications with [TestCafe](https://github.com/DevExpress/testcafe/). These extensions allow you to select page elements in a way that is native to Aurelia.
 
 ## Install
-`npm install testcafe-aurelia-selectors`
 
-## Usage
+```
+npm install testcafe-aurelia-selectors
+```
+
+## Selector extensions
+
+### byValueBind
+
+Selects an element whose `value` attribute is bound to the specified model property.
+
+```
+byValueBind(model, selector)
+```
+
+Parameter                   | Description
+--------------------------- | -----------
+model                       | The model property to which the element's `value` attribute is bound.
+selector&#160;*(optional)*  | A TestCafe [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). If specified, TestCafe will search for the target element among the descendants of the element identified by this selector.
 
+### byShowBind
+
+Selects an element whose `show` attribute is bound to the specified model property.
+
+```
+byShowBind(model, selector)
 ```
+
+Parameter                   | Description
+--------------------------- | -----------
+model                       | The model property to which the element's `show` attribute is bound.
+selector&#160;*(optional)*  | A TestCafe [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). If specified, TestCafe will search for the target element among the descendants of the element identified by this selector.
+
+### byCheckedBind
+
+Selects an element whose `checked` attribute is bound to the specified model property.
+
+```
+byCheckedBind(model, selector)
+```
+
+Parameter                   | Description
+--------------------------- | -----------
+model                       | The model property to which the element's `checked` attribute is bound.
+selector&#160;*(optional)*  | A TestCafe [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). If specified, TestCafe will search for the target element among the descendants of the element identified by this selector.
+
+### byFocusBind
+
+Selects an element whose `focus` attribute is bound to the specified model property.
+
+```
+byFocusBind(model, selector)
+```
+
+Parameter                   | Description
+--------------------------- | -----------
+model                       | The model property to which the element's `checked` attribute is bound.

```

Comment:
Element has not `focus` attribute.
When `model` is set to true, the input gets focus;
And vise versa